Formula/udp2raw-multiplatform.rb

```diff
@@ -0,0 +1,30 @@
+class Udp2rawMultiplatform < Formula
+  desc "Multi-platform(cross-platform) version of udp2raw-tunnel client"
+  homepage "https://github.com/wangyu-/udp2raw-multiplatform"
+  url "https://github.com/wangyu-/udp2raw-multiplatform/archive/refs/tags/20210111.0.tar.gz"
+  sha256 "712ad3c79b6ef5bf106c615823d0b0b3865d1c957f9838cf05c23b7ac7024438"
+  license "MIT"
+
+  depends_on "make" => :build
+  depends_on "libnet"
+  depends_on :macos
```

Comment:
```suggestion
```
This doesn't seem accurate. The name is `-multiplatform`, and there are Linux install instructions in their [wiki](https://github.com/wangyu-/udp2raw-multiplatform/wiki/How-to-build#for-linux-debian).

Reply:
removed